### DISCUSSION TITLE
Setable Gapless Playback

### DISCUSSION THREAD

**Discussion by nobaraos12** [ciderapp/Cider#1622] [Feature Request]
A setable gapless playback would be nice just like spotify has

**Comment by awsphire** [comment]
I was just about to ask the same thing. It would be great to be able to set the amount of seconds for the crossfade between two songs like in iTunes !

**Reply by nobaraos12** [reply]
yes it was a feature I heard cider had but they must have been referring to "seemless playback" which is not the same as gapless playback.

Gapless playback cuts the last 5 seconds of the end of the song and the first 5 of the start of the new song. If the user set 5 seconds. Typically gapless playback can be set to 3-15 seconds this would be great if cider adds this feature.
